.github/workflows/ruff_linter.yml

```diff
@@ -0,0 +1,12 @@
+name: Ruff
```

Comment:
Are we actually switching to ruff for all projects? We should probably discuss (briefly) at the next Infrastructure meeting.

Reply:
No, we're using black. don't switch this kind of thing please.

if you have a compelling case to switch, we should discuss, but then we'll switch all our projects.